### DISCUSSION TITLE
Java

### DISCUSSION THREAD

**Discussion by sowelioko** [holzschu/a-shell#733] [Ideas]
Is it possible to install a JDK on a-Shell, and if not, is there any other way I could compile Java on a-Shell?

If not, could this be a new feature?

**Comment by holzschu** [comment]
That looks challenging. I see there is OpenJDK, but cross-compiling that is a huge project.

**Reply by sowelioko** [reply]
I know iSH did it, but I don't make mobile apps so I don't know how.

- I was able to import (and run) OpenJDK through apk add, but I can't find the .bashrc file to add Java's path to $PATH automatically, so I'd really prefer to use a-Shell for this.
